### ISSUE TITLE
Financial documents after adding a new offerPart, refunding or exchanging the booking

### ISSUE THREAD

**Issue by Azurab** [UnionInternationalCheminsdeFer/OSDM#564]
Hi.

I have another question about general transaction flow. Is an API consumer obligated to POST bookings/{bookingId}/documents (subsequent) after confirming the booking change?

**Comment by jspetrak** [comment]
No, this is an optional flow when you want to obtain optional types of documents for booking, The ticket documents are Fulfillments under OSDM, and are generated automatically.

**Comment by CGantert345** [comment]
Documents are listed in the booking after the confirmation of the refund or exchange. The booking must be retrieved to get the documents.

**Comment by CGantert345** [comment]
TODO: add documentation in the general flow for refund / exchange

**Comment by Azurab** [comment]
My proposal of clarification to the after sale flow:
> Please be advised that, following the confirmation of the refund and exchange operation or addition of an offerPart to the booking, the distributor must update the fulfilments and documents (e.g. receipts) and the API consumer is required to take an action to retrieve the full booking to retrieve it.

**Comment by CGantert345** [comment]
TODO split sales and after-sales processes
TODO add intoduction to define the meaning of different after sales processes
